defaults/main.yml

```diff
@@ -14,12 +14,14 @@ git_remote_branch: origin
 git_remote_user: john
 git_remote_token: 1234567890
 git_remote_repo: nifi-registry
+git_remote_organization: tabajara-corp
```

Comment:
would be best to define its vars from the vars in the newoffers-infra ? I've looked into it and it looks like its is a private organization.